<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>455</width>
     <height>161</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>441</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>455</width>
     <height>220</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="rowCount">
    <number>4</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>55</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>90</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <column>
    <property name="text">
     <string>C1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>C2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>C3</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>C4</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>C5</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btn_random_number">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>9</y>
     <width>221</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Заполнить случ. числами</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_solve">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>78</y>
     <width>221</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выполнить задание</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_max_el">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Максимальный элемент:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_min_el">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>501</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Минимальный элемент:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_task_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>631</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Если максимальный элемент в таблице расположен после минимального, то &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_task_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>551</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;поменять значения мин. и макс. на 1, остальное на 0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_error">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>170</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>681</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
